pom.xml

```diff
@@ -1145,6 +1150,70 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <!--
+        the AWS module pulls in jackson; its transitive dependencies can create
+        intra-jackson-module version problems.
+        -->
+      <dependency>
```

Comment:
I'm declaring them here purely as declarations, so there's one place where all the hadoop dependencies go, versions declared etc. The cloud POM explicitly adds them to the module dependency, so that's the only place where they get pulled into the build. It's similar to what's done with Hive: spark-hive declares the actual dependencies, but the root pom declares what hive version to pull in, and what stuff to leave out.